libraries/stdlib/jvm/src/kotlin/collections/CollectionsJVM.kt

```diff
@@ -45,6 +45,14 @@ internal actual fun <T> Array<out T>.copyToArrayOfAny(isVarargs: Boolean): Array
     else
         java.util.Arrays.copyOf(this, this.size, Array<Any?>::class.java)
 
+@PublishedApi
+@SinceKotlin("1.3")
+@InlineOnly
+internal actual inline fun <T> List<T>.optimizeReadOnlyListCompat(): List<T> =
+    if (apiVersionIsAtLeast(1, 1, 0))
```

Comment:
I wasn't sure what version `optimizeReadOnlyList` was added. If you do, let me know and I'll change this check to the correct value.